bin/setup

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
```

Comment:
Although I don't use it, I still think it's a sensible default for apps/gems to have an initial setup script for the development environment. Among many others, see Factory Girl's [bin/setup](https://github.com/thoughtbot/factory_girl_rails/blob/master/bin/setup) as an example.